### PR TITLE
Branding ✨

### PR DESCRIPTION
**Pull request by rock3r**
## Problem

`no-player` doesn't have an identity...

## Solution

Enter @chris95x8 and his wizardry skills! He's done an awesome icon for the library. I've updated the README too, and the LICENSE too since it was a bit jumbled and GitHub misunderstood what it was (it thought it was ECL instead of Apache 2.0).

### Test(s) added 

No.

### Screenshots

noplayer-icon

### Paired with 

@chris95x8

### CODE REVIEW

**Comment by ataulm** [comment]
NIIICE

File: team-props/static-analysis/lint-config.xml [hunk 1]
```diff
@@ -19,4 +19,9 @@
     <ignore path="*generated/res/google-services/*" />
   </issue>
 
+  <!-- This issue flags the adaptive icons' background drawables too, which must be square and fully opaque -->
+  <issue id="IconLauncherShape" severity="error">
```
Comment (rock3r) [inline team-props/static-analysis/lint-config.xml]: Reported as Lint bug https://issuetracker.google.com/issues/70276438

**Comment by blundell** [comment]
https://en.wikipedia.org/wiki/Trypophobia

**Comment by rock3r** [comment]
LOL @blundell that wiki pic is discomforting very much to me but the icon here is fine :P

**Pull request merged**